docs/technical.rst

```diff
@@ -21,17 +21,28 @@ Runtime environment for development is `node.js <https://nodejs.org/en/>`_.
 Development should always be possible running the last two ``LTS`` Node.js versions,
 see Node.js `release schedule table <https://github.com/nodejs/Release#release-schedule>`_.
 
-====================== =================================================
-Node.js 8              Supported
-Node.js 10             Supported
-====================== =================================================
+====================== ================= ===============================
+Node Version           Status            Latest Status Change
+====================== ================= ===============================
+Node.js 8              Supported         2018-11-01
+Node.js 10             Supported         2018-11-01
+====================== ================= ===============================
 
 Node.js Features
 ^^^^^^^^^^^^^^^^
 
-``TODO``
+Buffer vs safe-buffer
+"""""""""""""""""""""
+Old Node versions up to versions ``4`` and ``5`` allowed for some unsafe usage
+of the ``Buffer`` API which led to the development of a replacement 
```

Comment:
It isn't really a replacement, but a polyfill. Maybe it was first, it is just nitpicking :)